packages/addons-manager/bundle/config/values.yaml

```diff
@@ -14,6 +14,7 @@ tanzuAddonsManager:
     tolerations: []
     webhookServerPort: 9865
     healthzPort: 18316
+    clusterDeleteTimeout: "5m"
```

Comment:
The default in the code is 10m. Lets stick to that

Reply:
done